src/js/components/Select/stories/ClearValue.js

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Clear } from 'grommet-icons/icons/Clear';
+
+import { Box, Grommet, Select, Button } from 'grommet';
+import { grommet } from 'grommet/themes';
+import { Text } from '../../Text';
+
+const optionList = [
+  'One',
+  'Two',
+  'Three',
+  'Four',
+  'Five',
+  'Six',
+  'Seven',
+  'Eight',
+  'Nine',
+  'Ten',
+];
+
+const UnSelect = () => {
+  const [value, setValue] = useState('');
+
+  const onClickClearOptions = () => {
+    setValue('');
+  };
+
+  return (
+    <Grommet theme={grommet}>
+      <Box
+        pad="medium"
+        direction="row"
+        align="center"
+        justify="center"
+        gap="small"
+      >
+        <Box>
+          <Text>Select with default clear selection</Text>
+          <Select
+            options={optionList}
+            value={value}
+            onChange={({ option }) => setValue(option)}
+            placeholder="Select a value"
+            clear
+            multiple
+          />
+        </Box>
+        <Box>
+          <Text>Select with custom location and label</Text>
+          <Select
+            options={optionList}
+            value={value}
+            onChange={({ option }) => setValue(option)}
+            placeholder="Select a value"
+            multiple
+            clear={{
+              position: 'bottom',
+              label: 'Unselect',
+            }}
+          />
+        </Box>
+        <Box>
+          <Text>Select with custom select render</Text>
+          <Select
+            options={optionList}
+            value={value}
+            onChange={({ option }) => setValue(option)}
+            placeholder="Select multiple options"
```

Comment:
It doesn't really allow you to select multiple options.

Reply:
sorry this must be a bad merge. are the other ones obselete as well?

Reply:
how come? on storybook i can select multple.

Reply:
Really? I went to the renderValue example, chose 'One' on Select, and then selected 'Two' and it automatically unselected the 'One' for me.
![image](https://user-images.githubusercontent.com/6320236/74769942-8aeafb80-5248-11ea-98af-4254dd8d3e0c.png)

What am i missing?

Reply:
lol weird i recalled i didi with a multiple hahaha. yeah lets remove it then.

Reply:
we should fix it.